resotocore/tests/resotocore/cli/command_test.py

```diff
@@ -892,16 +892,17 @@ async def test_pagerduty_alias(cli: CLI, echo_http_server: Tuple[int, List[Tuple
     # override timestamp
     assert response["payload"]["timestamp"] is not None
     response["payload"]["timestamp"] = "2023-02-10T15:03:33Z"
+    print(requests[0][1])
```

Comment:
Is this supposed to be there? Since its in a test I assume yes?

Reply:
No - I removed it 👍